src/common.browser/WebsocketMessageAdapter.ts

```diff
@@ -303,8 +303,13 @@ export class WebsocketMessageAdapter {
             }
 
             this.onEvent(new ConnectionMessageSentEvent(this.privConnectionId, new Date().toISOString(), sendItem.Message));
-            this.privWebsocketClient.send(sendItem.RawWebsocketMessage.payload);
+
+            // add a check for the ws readystate in order to stop the red console error 'WebSocket is already in CLOSING or CLOSED state' appearing
+            if (this.isWebsocketOpen) {
+                this.privWebsocketClient.send(sendItem.RawWebsocketMessage.payload);
+            }
```

Comment:
There's likely another bug at the root of getting that message and I worry about silently not sending a message.

Reply:
It'll return a PromiseHelper.fromError now if the socket isn't ready